Jellyfin.Xtream/Service/StreamService.cs

```diff
@@ -419,6 +419,7 @@ public MediaSourceInfo GetMediaSourceInfo(
                     ColorRange = videoInfo?.ColorRange,
                     ColorSpace = videoInfo?.ColorSpace,
                     ColorTransfer = videoInfo?.ColorTransfer,
+                    Width = videoInfo?.Width,
```

Comment:
Could you move the `Width` such that all properties are still sorted alphabetically?